oak_tensorflow_service/build.rs

```diff
@@ -14,7 +14,13 @@
 // limitations under the License.
 //
 
+use std::{path::PathBuf, process::Command};
+
 const SCHEMA: &str = "schema.fbs";
+const TFLITE_LIBRARY_NAME: &str = "tflite-micro";
+const TFLITE_BUILD_TARGET: &str = "//cc/tflite_micro:tflite-micro";
+// TODO(#3297): Get workspace path from environment variables.
+const TFLITE_LIBRARY_DIR: &str = "/workspace/bazel-bin/cc/tflite_micro/";
```

Comment:
You can use something like this

https://github.com/project-oak/oak/blob/ffb1cef36f28f58d568ce4a997dbf1c881c33804/oak_functions/sdk/oak_functions/tests/integration_test.rs#L28

Reply:
Done